pkg/controller/restore_controller_test.go

```diff
@@ -35,6 +35,71 @@ import (
 	. "github.com/heptio/ark/pkg/util/test"
 )
 
+func TestFetchRestore(t *testing.T) {
+	tests := []struct {
+		name             string
+		backupName       string
+		informerBackups  []*api.Backup
+		backupSvcBackups map[string][]*api.Backup
```

Comment:
I'd like to move to mocks instead of fake services, but we don't have to do it as part of this PR